lib/spack/spack/cmd/url.py

```diff
@@ -361,6 +380,21 @@ def print_stat(indent, name, stat_name=None):
         print_stat(4, git_type, "git_type")
     print_line()
 
+    if args.show_issues:
+        total_issues = sum(
+            len(issues)
+            for _, pkg_issues in issues.items()
+            for _, issues in pkg_issues.items()
+        )
+        print()
+        tty.msg("Found %d issues." % total_issues)
+        for issue_type, pkgs in issues.items():
+            tty.msg("Package URLs with %s" % issue_type)
+            for pkg, pkg_issues in pkgs.items():
+                color.cprint("    @*C{%s}" % pkg)
+                for issue in pkg_issues:
+                    print("      %s" % issue)
```

Comment:
Suggest use of the newer string formatting syntax (i.e., `format()`) instead of `%d` and `%s` formatting).

Reply:
Or better yet, `"    " + issue`

Reply:
Do y’all feel passionately about this?

Reply:
Not passionately. It's a style thing. Unfortunately no style tool has ever taken a stance on this so I can't really enforce it.

Reply:
I'm not seeing a huge win with the "new" old `format()` syntax.  f-strings make sense to me but IMO format() is no more readable than `%`.